doc/whats_new/v0.21.rst

```diff
@@ -54,11 +54,18 @@ Changelog
   :func:`datasets.fetch_covtype`,
   :func:`datasets.fetch_kddcup99`, :func:`datasets.fetch_olivetti_faces`,
   :func:`datasets.fetch_rcv1`, and :func:`datasets.fetch_species_distributions`
-  try to persist the previously cache using the new ``joblib`` if the cahced
+  try to persist the previously cache using the new ``joblib`` if the cached
   data was persisted using the deprecated ``sklearn.externals.joblib``. This
   behavior is set to be deprecated and removed in v0.23.
   :pr:`14197` by `Adrin Jalali`_.
 
+:mod:`sklearn.ensemble`
+.......................
+
+- |Fix| Fix zero division error in :func:`HistGradientBoostingClassifier` and
+  :func:`HistGradientBoostingRegressor`.
+  :pr:`14024` by `Nicolas Hug <NicolasHug>`.
```

Comment:
There was no what's new entry, but I think it's good to have one even if the estimators are experimental.